docs/Users_Guide/release-notes.rst

```diff
@@ -39,6 +39,53 @@ describes the bugfix, enhancement, or new feature:
 https://github.com/dtcenter/METplus/issues
 
 
+METplus Version 5.0.0 Beta 3 Release Notes (2022-09-21)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. warning:: **MAJOR CHANGES**:
+
+  * Use cases that include **EnsembleStat** wrapper will require config file updates. **NEW USER'S GUIDE SECTION OUTLINING REQUIRED CHANGES COMING SOON**
+  * The default value of :term:`SCRUB_STAGING_DIR` is now *True*.
+    This means some intermediate files that are auto-generated by METplus such as file lists and
+    uncompressed files will automatically be removed unless this option is set by the user.
+    These files are typically only used to debug unexpected issues.
+  * The default value of :term:`METPLUS_CONF` now includes the :term:`LOG_TIMESTAMP` so each METplus run
+    will generate a unique final config file, i.e. metplus_final.conf.20220921121733.
```

Comment:
```suggestion
    will generate a unique final config file, e.g. metplus_final.conf.20220921121733.
```
Minor point, but it seems like "for example" (e.g.) is a better fit here than "that is" (i.e.)